### DISCUSSION TITLE
Give png to LoadImaged

### DISCUSSION THREAD

**Discussion by JuAnYoO** [Project-MONAI/MONAI#6814] [Q&A] [answered]
`test_transforms = Compose(
    [
        LoadImaged(keys=["image1", "image2"]), 
        EnsureChannelFirstd(keys=["image1", "image2"]),
        EnsureTyped(keys=["image1", "image2"]),
        ScaleIntensityd(keys=["image1", "image2"], minv=0.0, maxv=1.0),
        Resized(
            keys=["image1", "image2"],
            spatial_size=(512, 512),
        ),
        CenterSpatialCropd(keys=["image1", "image2"],
                           roi_size=(448, 448),
                          ),          
        ToTensord(keys=["image1", "image2"]),
    ]
)`

I receive png image from user. Thus when I give dictionary with png(not an file path of png image) error occurs. How can I give png image to `LoadImaged` without error?

**Comment by KumoLiu** [comment, marked as answer]
Hi @JuAnYoO,

> Thus when I give dictionary with png(not an file path of png image) error occurs

I didn't get your point, what mean "dictionary with png(not an file path of png image)"?
I imaged that input should be:
```
[
{"image1": "image1_1.png", "image2": "image2_1.png"},
{"image1": "image1_2.png", "image2": "image2_2.png"}
....
]
```
Or if you have already read the png, then you can remove `LoadImaged`.

Thanks!

**Reply by JuAnYoO** [reply]
Thankyou!
